Entity/AbstractEntity.php

```diff
@@ -28,6 +28,16 @@ public function getCreated()
     }
 
     /**
+     * Set updated date
+     *
+     * @param \Datetime $date The updated date
+     */
+    public function setUpdated($date)
```

Comment:
Maybe configuration is missing ? I think we need to add [gedmo.listener.timestampable](https://github.com/Atlantic18/DoctrineExtensions/blob/master/doc/symfony2.md#doctrine-extension-listener-services) service in "services.yml" or "app/config/doctrine_extensions.yml"